client/src/components/Form/Elements/FormCheck.vue

```diff
@@ -60,7 +60,7 @@ watch(currentValue, () => {
             :indeterminate="indeterminate"
             @input="onSelectAll"
             @change="emit('indeterminate', selectAll)">
-            Select / Deselect all
+            Select / Unselect all
```

Comment:
This doesn't feel correct to me. I consider 'deselect' the action of removing selection, whereas 'unselected' is the state.

Reply:
You are the native speaker so I'd rather trust you on this but some Google search led me to a Quora article:

<img width="733" alt="image" src="https://user-images.githubusercontent.com/2105447/206726513-b1283989-5959-4afb-818d-92a92320af9d.png">

Reply:
Not sure about the quora article, but here's the dictionary:

![image](https://user-images.githubusercontent.com/155398/206726735-142f7be6-1828-48b3-9963-13b27cd40e64.png)

Reply:
Switched it to `Deselect all` to resolve this. Thanks for looking into it.